src/explore-education-statistics-common/src/modules/charts/util/createDataSetCategories.ts

```diff
@@ -338,10 +346,7 @@ export default function createDataSetCategories(
     })
     .filter(category => Object.values(category.dataSets).length > 0);
 
-  return sortDataSetCategories(dataSetCategories, axisConfiguration).slice(
-    axisConfiguration.min ?? 0,
-    (axisConfiguration.max ?? dataSetCategories.length) + 1,
-  );
+  return sortDataSetCategories(dataSetCategories, axisConfiguration);
```

Comment:
This function now also does the slicing of the min/max range. To make the function respect its name a bit more (it only implies that it sorts, rather than slices), would suggest renaming it to `getSortedDataSetCategoriesRange`